rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json

```diff
@@ -663,11 +663,15 @@
             "type": "keyword"
           },
           "preferred_name": {
-            "type": "text"
+            "type": "text",
+            "copy_to": "sort_preferred_name"
+          },
+          "sort_preferred_name": {
+            "type": "keyword"
           },
           "term": {
             "type": "text",
-            "copy_to": "facet_subjects",
+            "copy_to": ["facet_subjects", "sort_term"],
```

Comment:
Why do you create an new copy field with exact same settings than `facet_subjects` ? It's a little bit overkill to have 2 copy fields with same data, same analyzer, ...

Reply:
Changed.